### PR TITLE
build(deps): bump node-notifier from 8.0.0 to 9.0.0

### PR DESCRIPTION
**Pull request by Monster030**
Fixes named pipe in wsl2

- [x] passed all test

### CODE REVIEW

**Review by Gvozd — CHANGES_REQUESTED** [review]
not need `package-lock.json` for this repo
Please remove it

File: index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function findFirstDFS(compilation, key) {
   }
 }
 
-WebpackNotifierPlugin.prototype.compileEndOptions = function (stats) {
+WebpackNotifierPlugin.prototype.compileEndOptions = function compileEndOptions(stats) {
```
Comment (Gvozd) [inline index.js:30]: Are these changes important for this PR?

Comment (Monster030) [reply index.js:30]: No changes, just lint and dumb deps

**Comment by Gvozd** [comment]
Please squash first and third commit, and edit the commit message to remove the mention of `package-lock.json`
The commit history must remain clean

**Comment by Monster030** [comment]
Should be ready for merge.

**Review by Gvozd — APPROVED** [review]

**Comment by Gvozd** [comment]
@Monster030 Thanks
Changes published at v1.13.0

**Pull request merged**